airflow/www/views.py

```diff
@@ -2601,7 +2601,7 @@ def landing_times(self, session=None):
             x_points[task_id] = []
             for ti in tis:
                 ts = ti.execution_date
-                if dag.schedule_interval and dag.following_schedule(ts):
+                if dag.following_schedule(ts):
```

Comment:
`DAG.following_schedule()` returns None when `DAG.schedule_interval` is `None` anyway, so the `dag.schedule_interval` check is not needed.